test/Bin/GameTest/GameTestMain.cpp

```diff
@@ -29,13 +31,14 @@ class GameThread {
 
         initDataPath(options.dataPath);
 
-        _config = std::make_shared<GameConfig>("openenroth.ini");
+        _config = std::make_shared<GameConfig>("openenroth_test.ini");
         ResetTestConfig(_config.get());
         _game = GameFactory().CreateGame(_application.get(), _config);
     }
 
     ~GameThread() {
         EngineIocContainer::ResolveLogger()->setBaseLogger(nullptr);
+        std::filesystem::remove(MakeDataPath("openenroth_test.ini"));
```

Comment:
OK and I've just realized that you also don't need `MakeDataPath` here, because `GameConfig` constructor takes a full path, not data folder-relative path.